packages/editor-ui/src/components/Node.vue

```diff
@@ -990,7 +990,7 @@ export default defineComponent({
 				border-radius: 50px;
 
 				&.executing {
```

Comment:
@elsmr Do you get when this is used? I didi get it and couldn't test it

Reply:
It's set here: https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/Node.vue#L461

<img width="860" alt="image" src="https://github.com/n8n-io/n8n/assets/8850410/484d018b-e3c6-4b8d-94b4-28dce72319ea">